File: MODUL-1/python/PRAK0110-231081721007-RaymondHariyono.py
import math

print("diketahui : ")
alas = int(input ("alas= "))
tinggi = int(input ("tinggi= "))
sisiA = tinggi 
sisiB = int(math.sqrt(alas**2 + tinggi**2))
sisiC = alas

keliling = sisiA + sisiB + sisiC
luas = 0.5 * alas * tinggi

print("jawab : ")
print(f"sisi A = {sisiA} cm")
print(f"sisi B = {sisiB} cm")
print(f"sisi C = {sisiC} cm")
print(f"keliling = {keliling} cm")
print(f"luas = {luas:.0f} cm")

File: MODUL-0/MODUL-1/bahasa c/PRAK0110-231081721007-RaymondHariyono.c
#include <stdio.h>
#include <math.h>
int main(){

int alas, tinggi, sisiA, sisiB, sisiC;
printf("diketahui : \n");

printf("alas = ");
scanf("%d" ,&alas);

printf("tinggi = ");
scanf("%d" ,&tinggi);

sisiA = tinggi;
sisiB = sqrt(pow (alas , 2) + pow(tinggi , 2));
sisiC = alas;

int keliling = sisiA + sisiB + sisiC;
int luas = 0.5 * alas * tinggi;

printf("jawab : \n");
printf("Sisi A = %d cm \n" , sisiA);
printf("Sisi B = %d cm \n" , sisiB);
printf("Sisi C = %d cm \n" , sisiC);
printf("keliling %d cm \n" , keliling);
printf("luas %d cm " , luas);

return 0;
}